### PR TITLE
Update nodleprotocol from polkadotjs-app/master branch

### PR DESCRIPTION
**Pull request by Douglasacost**

### CODE REVIEW

**Comment by Douglasacost** [comment]
https://github.com/NodleCode/polkadot-js-apps/issues/27

File: .github/workflows/push-master.yml [hunk 1]
```diff
@@ -7,18 +7,88 @@ on:
 jobs:
```
Comment (ETeissonniere) [inline .github/workflows/push-master.yml:7]: I don't think we want to modify the github actions workflows here. Ours differs from Parity's

**Comment by aliXsed** [comment]
@Douglasacost Thank you for this PR. I made a mistake in my description of the ticket and asked for merging the master while what I meant was the latest release which is right now v0.125.1. So very sorry about that. I think it was also the reason that you had to fix some issues like removing some files with svg and png references in them, because potentially the issue was also in the upstream code and not yet fixed there. Apart from that your merge commit [here]([Merge branch 'pjs-master' into dot-master](https://github.com/NodleCode/polkadot-js-apps/pull/29/commits/be6b5c794d4c3e6f783b26a75556f94eb3fda430)) has become too large which could become an issue for the next time we want to make the same upgrade. 
Is it possible to redo this upgrade based on 0.125.1 and keep our own commits very short?

**Comment by Douglasacost** [comment]
> @Douglasacost Thank you for this PR. I made a mistake in my description of the ticket and asked for merging the master while what I meant was the latest release which is right now v0.125.1. So very sorry about that. I think it was also the reason that you had to fix some issues like removing some files with svg and png references in them, because potentially the issue was also in the upstream code and not yet fixed there. Apart from that your merge commit [here]([Merge branch 'pjs-master' into dot-master](https://github.com/NodleCode/polkadot-js-apps/pull/29/commits/be6b5c794d4c3e6f783b26a75556f94eb3fda430)) has become too large which could become an issue for the next time we want to make the same upgrade. Is it possible to redo this upgrade based on 0.125.1 and keep our own commits very short?

Hey Alex, no worries. Then, tomorrow will update this PR, the https://polkadot.js.org/apps/#/explorer is from the master branch though

**Comment by Douglasacost** [comment]
As this PR is arrange from polkadotjs-app master is not valid, instead we will take this one: https://github.com/NodleCode/polkadot-js-apps/pull/30

**Pull request closed without merging**